Cargo.lock

```diff
@@ -1969,9 +1969,9 @@ checksum = "e6d5a32815ae3f33302d95fdcb2ce17862f8c65363dcfd29360480ba1001fc9c"
 
 [[package]]
 name = "futures"
-version = "0.3.24"
+version = "0.3.23"
```

Comment:
Please revert this as there's a bug in 0.3.23. 🤣 https://github.com/risingwavelabs/risingwave/pull/5657#issuecomment-1272294934

Reply:
The scaling test will fail with 0.3.23